recursion/lesson_plan.md

```diff
@@ -38,16 +52,48 @@ About ___ hours
 
 ### Things to Remember
 - You can solve all recursion problems with a while loop and vice versa 
-        - While loop solutions usually run faster and take up less computer memory
 	- Recursion solutions are usually simpler to implement and easier to read
-- Recursive algorithms are often used to solve problems with the Tree data structure (for example, the DOM is a tree!)
+- Recursive algorithms are often used to solve problems with the Tree data structures (for example, the DOM is a tree!)
 
 ### Demonstration
 Instructor demonstrates in the video walkthrough an example of a recursive algorith in Javascript.
 
 ### Independent Practice  
 
-### Challenge
-Look at this solution to the [Tower of Hanoi](https://stackoverflow.com/questions/6947653/how-does-recursive-algorithm-work-for-towers-of-hanoi) problem and explain what's happening in the code.
+### Challenges
+
+
+#### [Challenge] - Factorial
+
+A factorial number is the concept in math by value of multiplying numbers by each consecutively smaller number. 
+
+For example, the value of `3` factorial, is `6`, which can be calculated by:
+
+```
+3 * 2 * 1
+```
+
+The value of `10` factorial, for example, can be calculated by:
+
+```
+10 * 9 * 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1
+``` 
+
+> Write a function that uses recursion to calculate the factorial of a number.
+
+#### [Challenge] - Fibonacci
```

Comment:
It would be helpful to add a diagram of how to calculate Fibonacci number's here, but volunteering ended before I found a good one!

Reply:
![image](https://user-images.githubusercontent.com/10985090/37302546-137d6f7a-25e9-11e8-999e-c57c342b8678.png) and 
![image](https://user-images.githubusercontent.com/10985090/37302606-3758dda8-25e9-11e8-85f5-d049f45eb625.png) and 
![image](https://user-images.githubusercontent.com/10985090/37302624-47e3ebc2-25e9-11e8-881f-93661e4e57e0.png) are some posibilities.

Reply:
I think the top two are super helpful!